### ISSUE TITLE
Vagrant setup returns error message on macOS Sierra

### ISSUE THREAD

**Issue by njyjn** [nusskylab/nusskylab#468]
`vagrant up` does not execute correctly on macOS Sierra, leaving error message

```
> vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'centos/7' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
The box 'centos/7' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/centos/7"]
Error:
```

Circumvent using the following:

1. `brew install curl`
2. `sudo mv /opt/vagrant/embedded/bin/curl /opt/vagrant/embedded/bin/curl_orig`
3. `sudo mv /opt/vagrant/embedded/bin/curl-config /opt/vagrant/embedded/bin/curl-config_orig`
4. `sudo ln -s /usr/bin/curl /opt/vagrant/embedded/bin/curl`
5. `sudo ln -s /usr/bin/curl-config /opt/vagrant/embedded/bin/curl-config`
6. `vagrant up`

Fix referenced from https://github.com/mitchellh/vagrant/issues/5016#issuecomment-260065012

Will be great to add into the README.md to inform incoming developers for a smoother setup.

**Comment by knmnyn** [comment]
Hi Justin:

Yes, when you pass the test, we'll ask you to add this yourself as a
feature.  Since you've made it an issue, it's well captured already.
Thanks!


Cheers,

Min

--
Min-Yen KAN (Dr) :: Associate Professor :: National University of Singapore
:: NUS School of Computing, AS6 05-12, 13 Computing Drive
Singapore 117417 :: +65 6516 1885(DID) :: +65 6779 4580 (Fax) ::
kanmy@comp.nus.edu.sg (E) :: www.comp.nus.edu.sg/~kanmy (W)

**Comment by njyjn** [comment]
@le-minhphuc and I have produced and tested instructions to bypass vagrant and run Skylab locally. Published to https://github.com/nusskylab/nusskylab/wiki/Local-Installations-(Non-Vagrant)

@wengkiat to fix Vagrant file.

**Comment by wengkiat** [comment]
Fixed in pull request #551
